generic_config_updater/generic_updater.py

```diff
@@ -36,6 +37,121 @@ def extract_scope(path):
     return scope, remainder
 
 
+def get_cmd_output(cmd):
+    proc = subprocess.Popen(cmd, text=True, stdout=subprocess.PIPE)
+    return proc.communicate()[0], proc.returncode
+
+
+def get_config_json_by_namespace(namespace):
```

Comment:
>get_config_json_by_namespace [](http://example.com/codeflow?start=4&length=28)

I see refactoring opportunity with "show runningconfiguration all" implementation. No need to implement it again.

Reply:
Is it possible to extract the core function underlying "show runningconfiguration all" which has no "bgpraw" at all? The point is that "bgpraw" in future may be not the only extra payload, and it will be awkward to add other fields there and also remove fields here.

Reply:
changed to original sonic-cfggen way to retrieve configs.